src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -3098,6 +3098,53 @@ public string AwsAccountId
         }
         #endregion
 
+        #region Otel Bridge
+
+        // TODO: configure the initial list of activity sources to include
+        private static readonly string[] DefaultIncludedActivitySources = ["NewRelic.Agent", "TestApp activity source"];
+
+        private List<string> _includedActivitySources;
+        public List<string> IncludedActivitySources
+        {
+            get
+            {
+                if (_includedActivitySources == null)
+                {
+
+                    var includedActivitySources = DefaultIncludedActivitySources.ToList();
```

Comment:
It might be better to do something like the following.
```suggestion
                    var includedActivitySources = new List<string>(DefaultIncludedActivitySources);
```

Reply:
I don't think there's any function difference in those two imnplementations.

Reply:
.ToLIst usually requires enumerating the IEnumerable, but the .ctor taking the collection can do a bulk copy. For smaller collection sizes it probably doesn't matter. But it may matter as the lists grow. That was also the reason I recommended using `AddRange` to combine the lists.